docs/cmake.md

```diff
@@ -174,7 +177,7 @@ OPENDDS_TARGET_SOURCES(publisher
 ```
 
 Another snippet, based upon [Messenger with auxiliary IDL library], showcases how an
-auxiliary IDL library can be created for inclusion by other executables:
+auxiliary (or dynamic) IDL library can be created for inclusion by other executables:
```

Comment:
I'm not sure I get what the intent is here.  The only distinction we're making is that IDL-generated code could be directly incorporated into a target that has user code, or it could be compiled into a separate library.  Whether or not that library is dynamic shouldn't be relevant.